index.html

```diff
@@ -1,4 +1,5 @@
 ---
 layout: index_layout
+permalink: /
```

Comment:
This is not necesarry

Reply:
I think it is? (I don't recall 100%) If e.g. in the navbar I click on the PhysiKon image without having this, the page that opens afterwards will be `physikon.pep-dortmund.org/index.html`. With this permalink it is just `physikon.pep-dortmund.org`.